contracts/src/v0.8/dev/functions/FunctionsOracle.sol

```diff
@@ -248,6 +249,7 @@ contract FunctionsOracle is
       } catch (bytes memory reason) {
         emit UserCallbackRawError(requestIds[i], reason);
       }
+      emit ResponseTransmitted(requestIds[i], transmitter);
```

Comment:
I have the same sentiment - fine with the changes, but can we do it without adding a new event?

Reply:
We can do this at the price of changing atlas EVM listener. Right now it is only watching for event logs, not for transactions. I suspect it will be hard to promote this functionality.
Alternatively, we can ask NOPs to share prometheus metrics with us, or we can ask them to set an alert on missing transmissions.
Let's take this conversation offline.